alpaca/trading/models.py

```diff
@@ -241,12 +241,50 @@ class Order(ModelWithID):
     hwm: Optional[str] = None
     position_intent: Optional[PositionIntent] = None
```

Comment:
How about adding `ratio_qty` here for the response of get/list orders?
```suggestion
    position_intent: Optional[PositionIntent] = None
    ratio_qty: Optional[Union[str, float]] = None
```